src/game/RandomizationOfShipsPositions.js

```diff
@@ -0,0 +1,152 @@
+import { SHIPS_LIST } from "./constants";
+
+const GAME_BOARD = document.querySelectorAll(".board__fields");
+
+let gameBoardArray = [];
+let fieldsNamesArr = [];
+let direction;
```

Comment:
It is not a mistake but for me, it is unneeded here. You don't export it or anything so it is needed only in your functions, which need it only for deciding which function should be used.

Reply:
I had on my mind the `let direction`. But maybe `let position` is also unneeded.